src/utils/status/host/constants.js

```diff
@@ -1,5 +1,74 @@
 export const HOST_STATUS_READY = 'HOST_STATUS_READY';
+export const HOST_STATUS_DISCOVERED = 'HOST_STATUS_DISCOVERED';
+export const HOST_STATUS_OK = 'HOST_STATUS_OK';
+export const HOST_STATUS_PROVISIONED = 'HOST_STATUS_PROVISIONED';
+export const HOST_STATUS_DEPROVISIONED = 'HOST_STATUS_DEPROVISIONED';
+export const HOST_STATUS_REGISTERING = 'HOST_STATUS_REGISTERING';
+export const HOST_STATUS_INSPECTING = 'HOST_STATUS_VALIDATION_ERROR';
+export const HOST_STATUS_PREPARING_TO_PROVISION = 'HOST_STATUS_PREPARING_TO_PROVISION';
+export const HOST_STATUS_PROVISIONING = 'HOST_STATUS_PROVISIONING';
+export const HOST_STATUS_DEPROVISIONING = 'HOST_STATUS_DEPROVISIONING';
+export const HOST_STATUS_MAKING_HOST_AVAILABLE = 'HOST_STATUS_MAKING_HOST_AVAILABLE';
+export const HOST_STATUS_VALIDATION_ERROR = 'HOST_STATUS_VALIDATION_ERROR';
+export const HOST_STATUS_REGISTRATION_ERROR = 'HOST_STATUS_REGISTRATION_ERROR';
+export const HOST_STATUS_STARTING_MAINTENANCE = 'HOST_STATUS_STARTING_MAINTENANCE';
+export const HOST_STATUS_STOPPING_MAINTENANCE = 'HOST_STATUS_STOPPING_MAINTENANCE';
+export const HOST_STATUS_MAINTENANCE = 'HOST_STATUS_MAINTENANCE';
+
+export const HOST_STATUS_ALL_ERROR = [HOST_STATUS_REGISTRATION_ERROR, HOST_STATUS_VALIDATION_ERROR];
 
-// TODO(honza): fill out these values
-export const HOST_STATUS_ALL_ERROR = [];
 export const HOST_STATUS_ALL_WARN = [];
+
+export const HOST_STATUS_ALL_PROCESS = [
+  HOST_STATUS_INSPECTING,
+  HOST_STATUS_PREPARING_TO_PROVISION,
+  HOST_STATUS_PREPARING_TO_PROVISION,
+  HOST_STATUS_PROVISIONING,
+  HOST_STATUS_DEPROVISIONING,
+  HOST_STATUS_MAKING_HOST_AVAILABLE,
+  HOST_STATUS_REGISTERING,
+  HOST_STATUS_STARTING_MAINTENANCE,
+  HOST_STATUS_STOPPING_MAINTENANCE,
+];
+
+export const HOST_STATUS_ALL_SUCCESS = [
+  HOST_STATUS_READY,
+  HOST_STATUS_DISCOVERED,
+  HOST_STATUS_OK,
+  HOST_STATUS_PROVISIONED,
+  HOST_STATUS_DEPROVISIONED,
+];
+
+export const RESPONSE_TO_HOST_STATUS = {
+  'validation error': HOST_STATUS_VALIDATION_ERROR,
+  inspecting: HOST_STATUS_INSPECTING,
+  discovered: HOST_STATUS_DISCOVERED,
+  OK: HOST_STATUS_OK,
+  ready: HOST_STATUS_READY,
+  'preparing to provision': HOST_STATUS_PREPARING_TO_PROVISION,
+  registering: HOST_STATUS_REGISTERING,
+  'registration error': HOST_STATUS_REGISTRATION_ERROR,
+  'making host available': HOST_STATUS_MAKING_HOST_AVAILABLE,
+  provisioning: HOST_STATUS_PROVISIONING,
+  deprovisioning: HOST_STATUS_DEPROVISIONING,
+  provisioned: HOST_STATUS_PROVISIONED,
+};
+
+export const HOST_STATUS_TO_TEXT = {
+  HOST_STATUS_READY: 'Ready',
```

Comment:
you should do this if you want to use the constant value: 

```js
{
  [HOST_STATUS_READY]: 'Ready',
}
```

Reply:
Done